dietstory-routing/game_version_updater/updateGameVersion.py

```diff
@@ -0,0 +1,36 @@
+from game_asset_manager.models import GameVersions
+from django.utils import timezone
+
+
+def add_all_jobs(job_scheduler):
+
```

Comment:
use a less generic name for this method

Reply:
Will we have more jobs for update GameVersions in the future?

Reply:
likely not but always a possibility